templates/social-template.svg

```diff
@@ -18,7 +18,7 @@
     <path d="M{{=it.widths[0]+6.5}} 6.5 l-3 3v1 l3 3" stroke="d5d5d5" fill="#fafafa"/>
   </g>
   {{?it.logo}}
-    <image x="5" y="3" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
+    <image x="5" y="2" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
```

Comment:
Could you include snapshots of how it looks on varying platforms to justify the change?